netlify/build.sh

```diff
@@ -46,7 +46,6 @@ function build() {
 
 # master builds skip the git clone and build the site in the current tree
 function build_master() {
-    echo "archive: true" >_config_jekyll.yml
     jekyll build --config $JEKYLL_CONFIG,$(pwd)/_config_jekyll.yml --baseurl /master --destination _site/master
```

Comment:
Since we're no longer creating _config_jekyll.yml, remove it from the build command
```suggestion
    jekyll build --config $JEKYLL_CONFIG --baseurl /master --destination _site/master
```